--- Net_SmartIRC-1.0.1/usr/share/pear/Net/SmartIRC.php~	2010-08-19 16:07:02.000000000 +0300
+++ Net_SmartIRC-1.0.1/usr/share/pear/Net/SmartIRC.php	2010-08-19 16:09:15.338384660 +0300
@@ -43,7 +43,7 @@
  *
  */
 // ------- PHP code ----------
-require_once 'SmartIRC/defines.php';
+require_once 'Net/SmartIRC/defines.php';
 define('SMARTIRC_VERSION', '1.1.0-dev ($Revision$)');
 define('SMARTIRC_VERSIONSTRING', 'Net_SmartIRC '.SMARTIRC_VERSION);
 
