--- Net_SmartIRC-0.5.1/SmartIRC.php~	Mon Jun 23 08:05:53 2003
+++ Net_SmartIRC-0.5.1/SmartIRC.php	Mon Jun 23 08:05:59 2003
@@ -36,8 +36,8 @@
  *
  */
 // ------- PHP code ----------
-include_once('SmartIRC/defines.php');
-include_once('SmartIRC/messagehandler.php');
+include_once('Net/SmartIRC/defines.php');
+include_once('Net/SmartIRC/messagehandler.php');
 define('SMARTIRC_VERSION', '0.5.1');
 define('SMARTIRC_VERSIONSTRING', 'Net_SmartIRC '.SMARTIRC_VERSION);
 
@@ -2536,4 +2536,4 @@
         unset($this);
     }
 }
-?>
\ No newline at end of file
+?>
