--- ./SmartIRC.php~	2005-09-29 03:21:19.000000000 +0300
+++ ./SmartIRC.php	2005-09-29 03:22:16.000000000 +0300
@@ -43,9 +43,9 @@
  *
  */
 // ------- PHP code ----------
-include_once('SmartIRC/defines.php');
-include_once('SmartIRC/irccommands.php');
-include_once('SmartIRC/messagehandler.php');
+include_once('Net/SmartIRC/defines.php');
+include_once('Net/SmartIRC/irccommands.php');
+include_once('Net/SmartIRC/messagehandler.php');
 define('SMARTIRC_VERSION', '1.0.0 ($Revision$)');
 define('SMARTIRC_VERSIONSTRING', 'Net_SmartIRC '.SMARTIRC_VERSION);
 
